Bill3.java

```diff
@@ -1,8 +1,14 @@
 // Splits a restaurant bill evenly among three diners.
 public class Bill3 {
 	public static void main(String[] args) {
+		
 		// To get you started, here is the first line in the program:
-	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+		int N = Integer.parseInt(args[3]);
```

Comment:
Names of variables should be indicative. This is for the whole question.. 
int total = Integer.parseInt(args[3]);